c/recovery.c

```diff
@@ -358,6 +362,66 @@ static void storageRelease(void *data, int size){
   );
 }
 
+#ifdef RCVR_CPOOL_STATES
+
+  #ifndef RCVR_STATE_POOL_SUBPOOL
+  #define RCVR_STATE_POOL_SUBPOOL 0
```

Comment:
I need to change this to 132, otherwise the recovery code doesn't obtain storage in the desired key when TCB key != PWS key.